src/transactions/test/wasm-go/test_invoke_contract.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+//export log_value
+func logValue(int64) int64
```

Comment:
Exports need to be named with `//export <function>` otherwise Go assumes they will have the name `main.<function>`. Since I was needing to provide the export notation, I figured I may as well appropriate case for Go code, otherwise I would not have bothered.